docs/api.md

```diff
@@ -133,6 +133,20 @@ custom visualizations. After a `collectOnly` run, the `ticks` data is stored in
 const ticks = JSON.parse(fs.readFileSync('{pid}.0x/ticks.json', 'utf8'))
 ```
 
+`opts` is an object with optional properties:
+
+ - `inlined` - data about inlined functions. 0x stores this in the `meta.json`
+   file, which you can read like:
+
+   ```js
+   const { inlined } = JSON.parse(fs.readFileSync('{pid}.0x/meta.json', 'utf8'))
```

Comment:
Suggestion: use the following instead:

```js
const { inlined } = require('{pid}.0x/meta.json')
```

Reply:
My preference is for the more verbose `fs`+`JSON` in this situation. in practice this will almost always be a dynamic file path. I'm not a big fan of recommending something like `require(path.join(pathTo0xData, 'meta.json'))`. To me, using fs and then JSON.parse looks like "I'm reading a data file" while `require` looks like "I'm importing a module".

Is there some best practice or benefit to using `require` that I'm not aware of? :smile:

Reply:
It's just shorter and less to read. So it's more intuitive for me. I guess it depends on how well you know the internals of require.

Reply:
I prefer `fs.readFileSync` as well. require caches things, while `fs.readFileSync` does not.